Formula/minidjvu.rb

```diff
@@ -30,13 +30,19 @@ class Minidjvu < Formula
   depends_on "djvulibre"
   depends_on "libtiff"
 
+  on_linux do
+    depends_on "gzip"
+  end
+
   def install
+    inreplace "Makefile.in", "/usr/bin/gzip", Formula["gzip"].opt_bin/"gzip" unless OS.mac?
+
     ENV.deparallelize
     # force detection of BSD mkdir
     system "autoreconf", "-vfi"
```

Comment:
This shouldn't be needed on Linux right?

Reply:
Indeed, it was unneeded to build on Linux, so I've added `if OS.mac?` to the end of that line.